Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -309,8 +312,7 @@ def render_GET(self, request):
 
         mc_community = self.get_multi_chain_community()
         nodes, edges = mc_community.get_graph(focus_node, neighbor_level)
-
-        return json.dumps({"focus_node": focus_node,
+        return json.dumps({"focus_node": hexlify(focus_node),
```

Comment:
Wouldn't the code look nicer if you were to just unhexlify in the mc_community call in 314 though? i.e. `mc_community.get_graph(unhexlify(focus_node), neighbor_level)`